src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,2 @@
+package core.basesyntax;public class Rectangle {
+}
```

Comment:
The class Rectangle is empty. According to the task, it should represent a geometric figure with appropriate properties and methods. Please implement the class according to the requirements.